extensions/2.0/Vendor/EXT_manifold/schema/glTF.EXT_manifold.schema.json

```diff
@@ -0,0 +1,48 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "title": "EXT_mesh_manifold extension",
+    "type": "object",
+    "description": "glTF extension defines manifoldness for a mesh.",
+    "allOf": [
+        {
+            "$ref": "glTFProperty.schema.json"
+        }
+    ],
+    "properties": {
+        "manifoldPrimitive": {
+            "allOf": [
+                {
+                    "$ref": "mesh.primitive.schema.json"
+                },
+                {
+                    "material": false
+                },
+                {
+                    "targets": false
```

Comment:
Sadly this doesn't seem to override these values in VSCode, for what it's worth.  After Googling a bit I'm not sure if it's even possible in a JSON schema to use `allOf` and then override some of the imported properties like this.  @lexaknyazev Do you know if one can import a schema (with `allOf` ... `$ref`) and then exclude two of the properties defined in the referenced schema?